omero/sysadmins/windows/server-installation.txt

```diff
@@ -328,120 +278,35 @@ is having a properly configured database.
 
 -  Create a non-superuser database user (make sure to note down the name
```

Comment:
The sentence at the start of this section can be reworded to remove the waffle.  "Probably the most important step towards having a working server system is having a properly configured database."

Let's remove it entirely or reword to e.g. "Create a database and user for OMERO as follows:"